web/themes/custom/drupalam/drupalam.theme

```diff
@@ -121,9 +121,8 @@ function drupalam_preprocess_image_formatter(&$vars) {
 }
 
 function drupalam_preprocess_node(&$variables) {
-
-  // Add comment count
-  if ($variables['node']->getType() == 'blog_post') {
+  // Add comment count.
+  if (in_array($variables['node']->getType(), ['blog_post', 'news'])) {
     $variables['comment_count'] = $variables['node']->get('field_blog_comments')->comment_count;
```

Comment:
Does News CT have `field_blog_comments `?

Reply:
Yes, I resued field_blog_comments field and styles for News.

Reply:
Well, not the best solution, but let it be that way for now.